jobs/ci-run/integration/integrationtests.yml

```diff
@@ -227,7 +227,7 @@
     project-type: "multijob"
     description: |-
       intergration-arm64 runs the new shell integration tests with arm64 workloads.
-    node: noop-parent-jobs
+    node: ephemeral-bionic-8c-32g-arm64
```

Comment:
My only concern is having a name mismatch in the future. But so far I think is OK.